docs/manual-instrumentation.md

```diff
@@ -18,11 +18,6 @@ or they might want to manually create spans for their own custom code.
 
 # Dependencies
 
-> :warning: prior to version 1.0.0, `opentelemetry-javaagent-all.jar`
-only supports manual instrumentation using the `opentelemetry-api` version with the same version
-number as the Java agent you are using. Starting with 1.0.0, the Java agent will start supporting
-multiple (1.0.0+) versions of `opentelemetry-api`.
-
```

Comment:
Are we sure about that? Have we tested that?

Reply:
we have a smoke test that verifies interop with opentelemetry-api 1.0.0, and #3044 to track doing better than that with muzzle

(also, this is old doc, I'm removing it, see https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/4106#discussion_r706967424)